codespell_lib/_codespell.py

```diff
@@ -642,6 +644,15 @@ def parse_file(filename, colors, summary, misspellings, exclude_lines,
         if line in exclude_lines:
             continue
 
+        extra_words_to_ignore = set()
+        match = inline_ignore_regex.search(line)
+        if match:
+            extra_words_to_ignore = set(
+                filter(None, (match.group("words") or "").split(","))
+            )
```

Comment:
Could use a set comprehension:
```suggestion
            extra_words_to_ignore = {w for w in (match.group("words") or "").split(",") if w}
```